public/assets/css/style.css

```diff
@@ -153,44 +159,49 @@ nav ul a {
     font-size: 2.8rem;
 }
 
-/* Data Viz */
-.chart-wrapper, .data-viz {
-    height: 100%;
-    display: flex;
-    flex-direction: column;
-    justify-content: space-between;
+.data-viz {
+    display: table;
+    width: 100%;
+    height: 65vh;
+    min-height: 0px;
 }
 
 #myChart {
     height: 80% !important;
     margin: 0 !important;
+    padding-top: 25px;
+    display: table-row;
```

Comment:
the display table and display table-row is what allows for a min-height of 0 px in the parent element. I tried this for slider but that needs the display block to work so no dice